exercises/acronym/lib/example.dart

```diff
@@ -0,0 +1,15 @@
+class Acronym {
+  String abbreviate(final String phrase) {
+    StringBuffer acronymBuilder = new StringBuffer();
+
+    RegExp expression = new RegExp(r'(^[A-z])|(\s[A-z])|-[A-z]');
+
+    List<Match> matchList = expression.allMatches(phrase).toList();
+
+    for (Match match in matchList) {
+      acronymBuilder.write(match.group(0).trim().toUpperCase().replaceAll("-", ""));
```

Comment:
If we were able to use the split to get rid of the characters that aren't alpha characters, the `replaceAll` may no longer be necessary.